pylintrc

```diff
@@ -23,7 +23,7 @@ max-locals=15
 # Maximum number of return / yield for function / method body
 max-returns=6
 # Maximum number of branch for function / method body
-max-branchs=12
+max-branches=12
```

Comment:
this one I've seen so many times already...

Reply:
Yeah, I'm was surprised it has not been fixed yet in NeuroM :thinking:

Reply:
It's fixed on v4 :)

Reply:
Ahhh that's it, that's why I thought it was already fixed :)